docs/getting-started/download.md

```diff
@@ -14,32 +14,37 @@ docker login ghcr.io
 
 ## Downloading the software
 
```

Comment:
suggestion, add `### HHFAB` here or similar title.  So the table of contents tracks the utilities that are being downloaded